locales/de.yaml

```diff
@@ -0,0 +1,275 @@
+  passwordChanged: Passwort erfolgreich geändert
+  successfullySignedIn: Erfolgreich eingeloggt
+  successfullySignedOut: Erfolgreich ausgeloggt
+  welcomeBack: "Schön dich wieder zu sehen, {firstName} !"
+  seeYouSoon: Bis bald!
+  editSuggestionsSent: Änderungsvorschläge abgeschickt und werden überprüft
+  editSuggestionApproved: Änderungsvorschlag erfolgreich angenommen
+  editSuggestionRejected: Änderungsvorschlag erfolgreich abgelehnt
+
+# COMMENT: NO NEED FOR MANUAL TRADUCTION -- admins will take care of it
+
+admin:
+  Admin: Admin
+  AdminMode: Admin mode
+  Approved: Approved
+  ApproveConfirmation: Approval confirmation
+  RejectConfirmation: Rejection confirmation
+  AreYouSureApprove: "Are you sure you want to approve {item}?"
+  AreYouSureReject: "Are you sure you want to reject {item}?"
+  AreYouSureDelete: "Are you sure you want to delete {item}?"
+  AreYouSureDeleteImage: Are you sure you want to delete this image?
+  CouldNotDelete: "Could not delete {item}"
+  CouldNotDeleteImage: Could not delete image
+  CouldNotUpdateEntry: Could not update entry
+  days:
+    Monday: Monday
+    Tuesday: Tueday
+    Wednesday: Wednesday
+    Thursday: Thuday
+    Friday: Friday
+    Saturday: Saturday
+    Sunday: Sunday
+  DeleteBodega: Delete Bodega
+  DeleteConfirmation: Delete confirmation
+  Deleted: Deleted
+  DragMarker: Drag the marker on the map
+  ImageSuccessfullyDeleted: Image successfully deleted
+  InStamstedersList: In Stamsteder's list
+  PleaseEnterName: Please enter a name
+  Save: Save
+  StreetHelp: Enter street name and number
+  SuccessfullyDeleted: "Successfully deleted {item}"
+  Suggestion: Suggestion
+  Suggestions: Suggestions
+  to: to
+  Users: Users
+
+# COMMENT:  MANUAL TRANSLATION REQUIRED -- pages meta data
+
+meta:
+  index:
+    title: Home
+    description: Discover Denmark's traditional pubs on an interactive map and find the ones closest to you !
+  about:
+    title: About
+    description: "A few words about the creators of the Bodega Map and the motivations behind it."
+  stats:
+    title: Statistics
+    description: "The Bodega database in numbers: percentage coverage of Denmark, number with specific characteristics, etc."
+  contact:
+    title: Contact
+    description: "Fill in this contact form and send some us some feedback. We'd love to hear what you think of Bodega Map"
+  login:
+    title: Login
+    description: "Log-in to the Bodega Map and get access to member-only features."
+  account:
+    title: My Account
+    description: See and edit your account's details for the Bodega Map
+  register:
+    title: Register
+    description: Create an account and enjoy the user-only features of the Bodega Map
+  verify:
+    title: E-mail verification
+    description: E-mail verification page for Bodega Map accounts
+  recovery:
+    title: Password recovery
+    description: Password recovery page for Bodega Map accounts
```

Comment:
Forgot to point out that this section also needs translation. Would you mind translating these last few strings? Thanks !